server/data/incidentReportingApi.ts

```diff
@@ -193,6 +193,31 @@ export type CorrectionRequest = {
   correctionRequestedAt: Date
 }
 
+export type NewIncident = {
+  type: string | string[]
+  incidentDateAndTime: string | string[]
+  prisonId: string | string[]
+  title: string | string[]
+  description: string | string[]
+  createNewEvent: boolean
+}
+
+export type UpdateIncident = {
+  incidentDateAndTime: string | string[]
+  prisonId: string | string[]
+  title: string | string[]
+  description: string | string[]
```

Comment:
so they are strings, and I set them to just be strings initially, but they come back as errors when trying to define them as the values that come back from the form are of type `string | string[]`. So that's why I have set them to be it. For now at least. But am open to a solution of how to fix that.

Reply:
I've sorted it just by explicitly forcing them to be strings on allocation

Reply:
cool beans! it's absolutely right to force the types on the _other_ side because these endpoints definitely do not accept arrays.